### DISCUSSION TITLE
DAI 2024-2025 - My picocli project - Louis Haye

### DISCUSSION THREAD

**Discussion by Artotrogo** [heig-vd-dai-course/heig-vd-dai-course#384] [Show and tell]
https://github.com/Artotrogo/DAI-Labo04/tree/main/java-intellij-idea-and-maven

**Comment by ludelafo** [comment]
Thank you! Everything seems good to me.

You can always find the solution in the solution repository if you ever need to compare your work with the official one here: https://github.com/heig-vd-dai-course/heig-vd-dai-course-solutions.

Do not hesitate to ask questions if something is not clear, either in this discussion or in the Q&A category of the GitHub Discussions here: https://github.com/orgs/heig-vd-dai-course/discussions/categories/q-a.
